packages/cli/src/models/initializer/ConfigVariablesInitializer.js

```diff
@@ -1,11 +1,14 @@
 import { ZWeb3, Contracts } from 'zos-lib'
 import Session from '../network/Session'
+import Compiler from '../compiler/Compiler'
 import TruffleConfig from './truffle/TruffleConfig'
 
 const ConfigVariablesInitializer = {
   initStaticConfiguration() {
     const buildDir = TruffleConfig.buildDir()
     Contracts.setLocalBuildDir(buildDir)
+    const solcSettings = TruffleConfig.solcSettings()
+    Compiler.setSettings(solcSettings)
```

Comment:
If truffle is not used, how does the user specify the compiler settings?

Reply:
That's an issue we have to solve somehow, already discuss that with @spalladino and we decided to tackle it later (see https://github.com/zeppelinos/zos/issues/502)